src/ripple/app/misc/Transaction.h

```diff
@@ -166,6 +301,12 @@ class Transaction
     TER             mResult = temUNCERTAIN;
     bool            mApplying = false;
 
+    /** different ways for transaction to be accepted */
+    AcceptedState   mAcceptedState;
+
+    /** current ledger state */
+    boost::optional<CurrentLedgerState> mCurrentLedgetState;
```

Comment:
The `m` prefix is a deprecated standard for indicating member variables. The new hotness standard is a `_` suffix. So instead of `mAcceptedState`, use `acceptedState_`, etc.

Reply:
Done